pkg/codegen/testing/test/testdata/output-funcs-edgeorder/nodejs/types/input.ts

```diff
@@ -9,29 +9,29 @@ import * as enums from "../types/enums";
 /**
  * Configuration filters
  */
-export interface ConfigurationFiltersArgs {
+export interface ConfigurationFilters {
```

Comment:
This reordering is suspect. I did not see any changes in node codegen that would cause it? We need to be deterministic otherwise it trips alarms in the providers team, when upgrades generate diffs.

Reply:
My first thought was Go's randomized map iteration order, but if you run this test on 1.19.x I expect you'll get this result every time.

I'm at a loss for why.

Reply:
Yeah! We might need to fixup the codegen to either do sorting or something to make sure this doesn't flop around.